### PR TITLE
Apiary - Remove all HTML and comments. Towards more Fabre-friendly

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -887,215 +712,16 @@ Response:
...
-
-+ Parameters
-    + entityId: Bcn_Welt (required, string) - Entity ID
-    + entityType: Room (required, string) - Entity Type
-    + attrName: temperature (required, string) - Attribute name
-
-+ Response 204
-
--->
-
```
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Maybe you have deleted too much and should have stopped here?

Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: yes, fixed at https://github.com/jmcanterafonseca/fiware-orion/commit/88d538aac3ddd69b907dc1480b71c38fb7828b73

**Comment by fgalan** [comment]
LGTM. Great!

Waiting for squash + rebase to merge

**Comment by fgalan** [comment]
LGTM

**Pull request merged**